nbgrader/docs/source/user_guide/advanced.rst

```diff
@@ -11,71 +11,6 @@ Running nbgrader with JupyterHub
 
 Please see :doc:`/configuration/jupyterhub_config`.
 
-.. _assignment-list-installation:
```

Comment:
I don't want to remove this documentation, as I think it is still a valid question someone might search for. Rather than deleting it, we should update it with the new installation instructions (i.e. with `--system`?)